configuration/yaml.md

```diff
@@ -0,0 +1,642 @@
+
+  TV_Volume:
+    type: Dimmer
+    label: Volume [%d]
+    channel: lgwebos:WebOSTV:3aab9eea-953b-4272-bdbd-f0cd0ecf4a46:volume
+```
+
+## Object Configuration Structure
+
+### Tags
+
+openHAB comes with pre-defined [semantic tags](https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.semantics/model/SemanticTags.csv) that can be readily used.
+It is also possible to add custom semantic tags into the system and subsequently use them alongside the built-in tags.
+
+Custom semantic tags can be defined under the `tags` section.
+
+#### Tags YAML Definition Syntax
+
+```yaml
+tags:
+  <tag_full_uid>:
+    label: <tag label>
+    description: <tag description>
+    synonyms:
+      - tag synonym 1
+      - tag synonym 2
+```
+
+Each tag is specified with the tag's full UID as the key.
+The rest are optional.
+
+Example:
+
+```yaml
+version: 1
+
+tags:
+  Location_Indoor_Room_HomeCinemaRoom:
+    label: Home Cinema Room
+    description: The room containing my home cinema system
+    synonyms:
+      - Home theater room
+
+  Location_Outdoor_VegetableGarden:
+    label: Vegetable Garden
+
+  Property_Height: {} # no label/description/synonyms
+```
+
+The tag's full UID defines the location of the semantic tag in the tree.
+In the example, `Location_Indoor_Room_HomeCinemaRoom` defines semantic tag `HomeCinemaRoom` as a child of `Room` which is a child of `Indoor` which is a child of `Location`.
+There are constraints on the name of a semantic tag (like HomeCinemaRoom): it must start with a capital letter and contain only letters and numbers.
+
+You can have multiple YAML files with different semantic tags but keep semantic tags dependent on each other in the same file.
+
+### Things
+
+The `things` top-level key contains a map of Things, defined by their Thing UID.
+
+#### Thing YAML definition syntax:
+
+```yaml
+things:
+  <thing_uid>:
+    isBridge: true|false
+    bridge: <bridge_uid>
+    label: <Thing label>
+    location: <Location>
+    config:
+      <config key>: <config value>
+    channels: # To define custom channels if necessary
+      <channel_id>:
+        type: <channel type>
+        kind: state|trigger # Default: state
+        itemType: <item type>
+        itemDimension: <item dimension> # Only if itemType is "Number"
+        label: <channel label>
+        description: <description>
+        config:
+          <channel config key>: <config value>
+```
+
+| Configuration Key | Description                                                                                                                                   |
+|:------------------|:----------------------------------------------------------------------------------------------------------------------------------------------|
+| thing_uid         | The UID of the Thing or Bridge being defined. The UID format is `binding_id`:`thing_type_id`:[`bridge_id`:]`thing_id`, e.g. `astro:sun:home`. |
```

Comment:
How about `<binding_id>:>thing_type_id>:[<bridge_id>:]<thing_id>`. The many changes in style look strange.
I think it would also be better if its `<thing_uid>` instead of "thing_uid"

Reply:
Thanks, I've changed it as suggested.